File: SoleusHotelApi/SoleusHotelApi/Helpers/RoomRequestParams.cs
using SoleusHotelApi.Enums;

namespace SoleusHotelApi.Helpers
{
    public class RoomRequestParams : PaginationParams
    {
        public RoomRequestStatus RoomRequestStatus { get; set; }
        public string Room { get; set; }
        public string Department { get; set; }
        public string Topic { get; set; }
        public string Subject { get; set; }
        public DateTime RequestDate { get; set; }
}
}

File: soleusHotelClient/src/app/core/models/roomRequestParams.ts
export class RoomRequestParams {
  pageNumber = 1;
  pageSize = 20;

  room: string;
  department: string;
  topic: string;
  subject: string;
  minRequestDate: Date;
  maxRequestDate: Date;
  requestStatus: string;
  minDateStart: Date;
  maxDateStart: Date;
  minDateEnd: Date;
  maxDateEnd: Date;
  assignedTo: string;
  orderBy = 'date';
}
